tools/image_processing/imagej2/imagej2_shadows/imagej2_shadows.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<tool id="imagej2_shadows" name="Add shadow effect" version="@WRAPPER_VERSION@.0">
+    <description>to images with ImageJ2</description>
```

Comment:
@bgruening This is another one of this items I've wondered about.  I put the "with ImageJ2" string in the initial few tools to make more visible what the wrapper is based on, and, of course, it has just snowballed.  I would happily eliminate this string from all tools though if you think it would be better.